docs/pages/enroll-resources/workload-identity/workload-attestation.mdx

```diff
@@ -37,11 +37,25 @@ available to be used when configuring rules for `tbot`'s Workload API service:
 
 | Field             | Description                                                                  |
 |-------------------|------------------------------------------------------------------------------|
-| `unix.attested` | Indicates that the workload has been attested by the Unix Workload Attestor. |
+| `unix.attested`   | Indicates that the workload has been attested by the Unix Workload Attestor. |
 | `unix.pid`        | The process ID of the attested workload.                                     |
 | `unix.uid`        | The effective user ID of the attested workload.                              |
 | `unix.gid`        | The effective primary group ID of the attested workload.                     |
 
+### Support for non-standard /proc mounting
+
+To resolve information about a process from the PID, the Unix Workload Attestor
+reads information from the `/proc` filesystem.
+
+In some cases, the `/proc` filesystem may not be mounted at the default
```

Comment:
Should we say "procfs" insted of "/proc filesystem"?

procfs is the accurate name for this technology and holds true even when not mounted at /proc

Reply:
Yup - that's definitely more accurate. I'll update the PR on monday before merging.

Reply:
[7688566](https://github.com/gravitational/teleport/pull/50217/commits/7688566d7fe3ad002c885d18849fae0268e57481)